TTS/trainer.py

```diff
@@ -151,19 +152,36 @@ def __init__(
 
         if config is None:
             # parse config from console arguments
-            config, output_path, _, c_logger, tb_logger = process_args(args)
+            config, output_path, _, c_logger, dashboard_logger = process_args(args)
 
         self.output_path = output_path
         self.args = args
         self.config = config
 
         # init loggers
         self.c_logger = ConsoleLogger() if c_logger is None else c_logger
-        if tb_logger is None:
-            self.tb_logger = TensorboardLogger(output_path, model_name=config.model)
-            self.tb_logger.tb_add_text("model-config", f"<pre>{config.to_json()}</pre>", 0)
-        else:
-            self.tb_logger = tb_logger
+        self.dashboard_logger = dashboard_logger
```

Comment:
Maybe you can put this part in a function and place it under TTS/utils/logging/__init__.py, as it is being called in multiple places.

Reply:
Yes makes sense. I'll do it :)

Reply:
Done